File: TeemoAttacking/TeemoAttacking.py
class Solution:
    def findPoisonedDuration(self, timeSeries: List[int], duration: int) -> int:
        var = 0
        for i in range(0, len(timeSeries) - 1):
            var += min(timeSeries[i + 1] - timeSeries[i], duration)
        return var + duration

File: TeemoAttacking/TeemoAttacking.js
/**
 * @param {number[]} timeSeries
 * @param {number} duration
 * @return {number}
 */
var findPoisonedDuration = function (timeSeries, duration) {
  let bro = 0;
  for (i = 0; i < timeSeries.length - 1; i++) {
    bro += Math.min(timeSeries[i + 1] - timeSeries[i], duration);
  }
  return bro + duration;
};
